Libraries/CMSIS/Device/Maxim/MAX32672/Include/usr_aeskeys_regs.h

```diff
@@ -76,6 +77,7 @@ extern "C" {
 
 /**
  * @ingroup     usr_aeskeys
+ * @ingroup     aes
```

Comment:
Should the @ingroup annotation for all the aeskeys be `aes`?

Reply:
Any `@ingroup` will be listed like this:

![image](https://user-images.githubusercontent.com/38844790/230989584-173900fa-76f8-49e4-b038-60e2c3b19cba.png)

So I think a good criteria for what gets the "root" `aes` group is to match exactly what is included by the peripheral's header file.

Ex "aes.h":
![image](https://user-images.githubusercontent.com/38844790/230989895-1448c4a9-7e2e-48a4-9da8-15ba2345116b.png)

Then the Doxygen browser will also give you clickable links that match 1:1 with its source code.

![image](https://user-images.githubusercontent.com/38844790/230990007-4287f804-3b48-4389-bcf1-139a9fcb469c.png)